src/main/java/modtrekt/logic/commands/ListModulesCommand.java

```diff
@@ -29,6 +30,12 @@ public ListModulesCommand(boolean areDoneTasksShown) {
 
     @Override
     public CommandResult execute(Model model) throws CommandException {
+        ModCode previousCode = model.getCurrentModule();
+
+        if (previousCode != null) {
+            throw new CommandException("Please exit current module first!");
```

Comment:
I think we should also notify the user of the command they should use to do this, e.g. `... with cd ..`.